.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     'object-shorthand': 'error',
     'prefer-const': 'error',
     'space-before-function-paren': ['error', 'always'],
+    'no-multi-spaces': 'error',
```

Comment:
can you put in alphabetic order with no-extra-parens ?
This makes easier to identify such options.

Reply:
Yeah. Didn't see it.